data-service/src/main/java/com/amazonaws/blox/dataservice/storage/model/DeploymentDDBRecordTranslator.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.dataservice.storage.model;
+
+import com.amazonaws.blox.dataservice.model.Deployment;
+
+public class DeploymentDDBRecordTranslator {
```

Comment:
I think I'll go through and fix this for everything I have so far and not just this PR cause I'll need to change it for create environment and get environment API PR as well.